Runtime/Provider/FileTextToSpeechProvider.cs

```diff
@@ -52,7 +52,7 @@ public async Task<AudioClip> ConvertTextToSpeech(string text)
             {
                 audioClip = await FallbackProvider.ConvertTextToSpeech(text);
 
-                if (Configuration.SaveAudioFilesToStreamingAssets)
+                if (Configuration.UseCache && (Configuration.StillWriteCacheWhenProjectIsBuilt || Application.isEditor))
```

Comment:
`StillWriteCacheWhenProjectIsBuilt` sounds like "Write cache when Unity builds the project even if Use Cache is toggled off". "UseCacheInBuiltApp" sounds better. Also, we can check for the current OS/build target on our own, so we could drop this setting, too.